test-cases/longevity/longevity-1TB-5days-authorization-and-tls-ssl.yaml

```diff
@@ -12,10 +12,11 @@ stress_cmd: ["cassandra-stress mixed cl=QUORUM duration=6800m -schema 'replicati
 run_fullscan: ['{"mode": "table_and_aggregate", "ks_cf": "random", "interval": 120}']
 round_robin: true
 
-n_db_nodes: 4
+n_db_nodes: 6
```

Comment:
Here we used 4 nodes because we wanted to make sure that 1TB dataset can fit into 4 x 4xlarge and whenever we reached out of space we knew something unexpected happened.

You can call it the old "90%" verification.
Increasing it to 6 without increasing the dataset will take some/most of the sting out of this test.
I don't have a good recommendation without testing it, but maybe now with the higher bar we should try it with 3 nodes.

Reply:
There were many improvements to disk utilization, so we can see if 3 nodes, gonna change this, thanks